setup.py

```diff
@@ -45,7 +45,7 @@ def load_requirements(*requirements_paths):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
     ],
-    packages=find_packages(exclude=["test", "tests"]),
+    packages=find_packages(exclude=['*.test', '*.tests']),
```

Comment:
From the second answer on http://stackoverflow.com/questions/8556996/setuptools-troubles-excluding-packages-including-data-files, I learned that `setuptools` uses `fnmatchcase`, and was able to interactively run `find_packages()` until I found the proper values for `exclude`.